applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -195,6 +204,10 @@ impl AppState {
             None
         }
     }
+
+    pub fn get_connected_peers(&self) -> &Vec<Peer> {
```

Comment:
nit: (thought this would be a 📎 , guess not)
```suggestion
    pub fn get_connected_peers(&self) -> &[Peer] {
```

Reply:
Don't really see the benefit here? Makes it clear what collection is being stored inside and its returning a reference not a clone.

Reply:
https://rust-lang.github.io/rust-clippy/master/#ptr_arg

Reply:
That lint is for arguments not returns. 🤷‍♂️ really don't see the benefit of hiding the fact that it's a vec.

Reply:
There is no benefit either way - it's consistency - it doesnt matter